File: CSE-206--AlgorithmsLab/Lab-01__2021-02-08__graph/Task__01__2dArray.java
import java.util.Scanner;
import java.util.Arrays;
public class Task__01__2dArray {
    public static void main(String[] args) {
        new Graph ();
    }
}
class Graph {
    Scanner input = new Scanner (System.in);
    int size;
    int matrix[][];
    Graph() {
        System.out.printf("Enter Size: ");
        this.size = input.nextInt();
        this.matrix = new int[size][size];
        graphInput();
        graphPrint();
    }
    void graphInput() {
        for (int r=0; r<size; r++) {
            for (int c=0; c<size; c++) {
                matrix[r][c] = input.nextInt();
            }
        }
    }
    void graphPrint() {
        System.out.println("AdjacencyMatrix: ");
        for (int r=0; r<size; r++) {
            for (int c=0; c<size; c++) {
                System.out.printf("%d", matrix[r][c]);
                if (c+1 == size) {
                    System.out.println();
                }
                else {
                    System.out.printf(" ");
                }
            }
        }
    }
}

File: CSE-206--AlgorithmsLab/Lab-01__2021-02-08__graph/Python/Task__01__2dArray.py
size = int(input("Size: "))
matrix = []
for r in range(size):
    tempMatrix = []
    for c in range(size):
        tempMatrix.append(int(input()))
    matrix.append(tempMatrix)
print("AdjacencyMatrix: ")
for r in range(size):
    for c in range(size):
        if c+1 != size:
            print(matrix[r][c], end = " ")
        else:
            print(matrix[r][c])
